<launch>
    <node pkg="dv_ros_capture" type="capture_node" name="capture_node" clear_params="true" output="screen"
          required="true">
        <rosparam param="unbiasedImuData">false</rosparam>
        <rosparam param="transformImuToCameraFrame">false</rosparam>
<!--        <rosparam param="cameraCalibrationFilePath">/home/vincenzo/dv_calibration_2022-04-22T09-32-29Z/calibration.json</rosparam>-->
    </node>

    <node pkg="dv_ros_imu_bias" type="bias_estimation_node" name="bias_estimation_node" clear_params="true" output="screen"
          required="false">
        <remap from="/bias_estimation_node/imu" to="/capture_node/imu"/>
    </node>

    <node pkg="rqt_gui" type="rqt_gui" name="gui"
          args="--perspective-file $(find dv_ros_capture)/config/dynamic_reconfigure_rqt.perspective"/>
</launch>